vars/test.groovy

```diff
@@ -1,5 +1,7 @@
 // public
 def lintHelm(chartName) {
+  helm.addHelmRepo('ffc', 'https://raw.githubusercontent.com/defra/ffc-helm-repository/master/')
```

Comment:
Would this URL not be better stored as a Jenkins env var so we can change it if necessary without a code update?